### ISSUE TITLE
What date format is expected/preferred for a dataset's collection date?

### ISSUE THREAD

**Issue by PGijsbers** [openml/OpenML#1035]
See title. OpenML datasets have a collection date, which is to be provided in a string. The [Python example](https://openml.github.io/openml-python/master/examples/30_extended/create_upload_tutorial.html#sphx-glr-examples-30-extended-create-upload-tutorial-py)  is ambiguous between MM-DD-YYYY or DD-MM-YYYY. The [schema definition](https://www.openml.org/api/v1/xsd/openml.data.upload) also does not specify the expected format.

**Comment by joaquinvanschoren** [comment]
I don't think this is enforced. I would say we choose a format and include it in the schema?

**Comment by PGijsbers** [comment]
In that case I would propose to use the YYYY-MM-DD format as per [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).

**Comment by joaquinvanschoren** [comment]
Agreed. The database stores datetime as 2016-01-22 10:26:38
